labonneboite/common/pagination.py

```diff
@@ -110,7 +110,7 @@ def get_current_page(self):
 
     def get_page_count(self):
         return 1 + \
-            (self.company_count - 1) / OFFICES_PER_PAGE
+            (self.company_count - 1) // OFFICES_PER_PAGE
```

Comment:
TIL https://stackoverflow.com/questions/1535596/what-is-the-reason-for-having-in-python